doc/adrs/109_add_support_for_image_overlay.adoc

```diff
@@ -0,0 +1,34 @@
+= ADR-109 - Add support for image overlay
+
+== Context
+
+Despite the fact that more than one image can be contribute for an EMF label, only the first one is displayed on the UI.
+So it's not possible for a specifier to build icons composed of several layers.
+
+== Decision
+
+* Change the `IObjectService` methode `getImagePath` to no longer return a single _string_ but the list of all the image paths.
```

Comment:
We talked about it and we agreed on changing `IObjectService#getImagePath` to return a list of strings, so why is there a new method named `IObjectService#getImagePathOverlay` instead. The goal was explicitly to have the change in `IObjectService#getImagePath`.

This contribution is not following what is described in the ADR.

Reply:
I introduced the `getImagePathOverlay ` to avoid to impact all the `IObjectService#getImagePath` consumers, only for the duration of a test on the tree.
Now I applied the change for all the iconURL I removed it.